src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        try (BufferedReader br = new BufferedReader(new java.io.FileReader(filePath))) {
```

Comment:
According to the checklist, we should use try-with-resources when working with classes that implement AutoCloseable, which is correctly done here. However, the BufferedReader can be improved by using `Files.newBufferedReader(Paths.get(filePath))` from `java.nio.file.Files` and `java.nio.file.Paths` for better handling of file paths and character encoding.